package.json

```diff
@@ -9,31 +9,22 @@
   "author": "João Pinho <jpe.pinho@gmail.com>",
   "contributors": [
     "João Pinho(https://github.com/jpinho)",
-    "Diogo Marques(https://github.com/JoFont)"
+    "Diogo Marques(https://github.com/JoFont)",
+    "Alexandre Marques(https://github.com/alexmarqs)"
```

Comment:
To deserve your name here you need to kiss the ring first 💍 😆

Reply:
ahah done 💍